k8s/configs/prometheus/values.yaml

```diff
@@ -118,6 +118,10 @@ grafana:
         datasource: Prometheus
         gnetId: 17092
         revision: 7
+      Kong:
```

Comment:
I believe this one is something extra:)

Reply:
Yep, and I went full panic mode when I realized it